src/context/TopologyPageContext.tsx

```diff
@@ -1,5 +1,3 @@
-import React, { useState, createContext, useContext } from "react";
```

Comment:
Why are we removing this ?

Reply:
It's no longer being used anywhere, we currently use React Query for caching purposes.